_posts/2021-12-06-announcing-applications-for-ols-5.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: OLS-4 Graduations and applications open for OLS-5!
+authors:
+- bebatut
+- emmyft
+- malvikasharan
+- yochannah
+image: https://images.unsplash.com/photo-1519070994522-88c6b756330e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1597&q=80
+photos:
+  name: Photo by Honey Yanibel Minaya Cruz on Unsplash
+  url: https://unsplash.com/@noneyyanibel
+---
+
+OLS-4 is approaching its last cohort call, which means that we are preparing for the graduation of our current cohort 
+and setting things up for the OLS-5 in 2022.
+
+## Graduations - you're all invited!
+
+Our amazing project leads for OLS-4 brought over 20 projects from around the world and have been working hard amid the pandemic to advance their knowledge in open and reproducible research practices while establishing themselves 
+as experts in their fields ([see our projects and participants]({% link _ols-4/projects-participants.md %}). 
+We are lucky enough to witness their projects and network grow while working alongside our mentors, expert speakers, 
+and call facilitators.
+
+It's time to set the stage for them to share their projects with the wider open science communities through our graduation calls on 18, 19 and 20 January 2022.
+We invite you to come and watch the [live-streamed calls on YouTube]({{site.youtube}}) - they'll be spread across three different 
+hour-long sessions, and available on YouTube after the event if you can't attend in real-time. 
+
+These calls will be hosted for the OLS members via Zoom and livestreamed for everyone on YouTube with real-time captions using Otter.ai.
+- Slot 1: 18 January, Tuesday - 09:00 - 10:00 AM UTC
+    - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-18/09:00/week-16-ols-4-grad1))
+- Slot 2: 19 January, Wednesday - 14:00 - 15:00 UTC
+    - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-19/14:00/week-16-ols-4-grad1))
+- Slot 3: 20 January, Thursday - 17:00 - 18:00 UTC
+    - ⏰ [In your timezone]((https://arewemeetingyet.com/London/2022-01-20/17:00/week-16-ols-4-grad1))
+
+We especially welcome you if you are:
+- friends, family, colleagues, of the current project leaders who are graduating.
+- planning to apply for OLS-5 and want to see what our current projects are like.
+- interested in learning about the OLS and what we do.
+- from an open research institute, funder, journal, or anyone else who would like to see some of the shining stars of early open science projects from this cohort.
+
+## Call for application for OLS-5 is NOW OPEN!
+
+Interested in applying for the next round? We can't wait to see your application! 
+Please submit your application via [OpenReview](https://openreview.net/group?id=openlifesci.org/Open_Life_Science/2022/Cohort_5) before the end of the day on **15 January 2022** (hard deadline).
```

Comment:
Let's just make sure not to surprise people w/ open review deadline for registration. 

```suggestion
Please submit your application via [OpenReview](https://openreview.net/group?id=openlifesci.org/Open_Life_Science/2022/Cohort_5) before the end of the day on **15 January 2022** (hard deadline). **Important note:** The deadline for registering for OpenReview is 12 January, as all OpenReview accounts are approved manually. 
```